sconstruct

```diff
@@ -314,10 +314,7 @@ def NVDADistGenerator(target, source, env, for_signature):
 		buildCmd.append("-O")
 	# Issue errors about str(bytes_instance), str(bytearray_instance)
 	buildCmd.append("-bb")
-	buildCmd.extend(("setup.py", "-vv", "build", "--build-base", buildDir.abspath,
-		"py2exe", "--dist-dir", target[0].abspath))
-	if release:

```

Comment:
Shouldn't this block be restored before merging?

Reply:
No, we don't need the optimize flag. 0d6970b set the optimization level to 2 (highest) for all builds in the py2exe options